src/garage/envs/half_cheetah_dir_env.py

```diff
@@ -54,9 +53,8 @@ def step(self, action):
                         control cost.
                     * reward_ctrl (float): The reward for acting i.e. the
                         control cost (always negative).
-                    * task (dict):
-                        * direction (float): 1.0 for forwards, -1.0 for
-                            backwards.
+                    * task_dir (float): Target direction. 1.0 for forwards,
```

Comment:
should this be an int or bool instead?

Reply:
I *think* it's easier for it to be a float.

Really, I feel like task out to be an `np.ndarray`, and we should also have a `task_space` containing it.
But this is fine.

Reply:
With new API (still under discussion), task would probably become a TaskSpec? @ryanjulian @naeioi

Reply:
Let's see how that API does in metaworld before we start propagating it.